core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -309,8 +309,13 @@ public boolean mkdirs(String path, MkdirsOptions options) throws IOException {
   }
 
   @Override
+  public InputStream open(String path, OpenOptions options) throws IOException {
+    return openObject(stripPrefixIfPresent(path), options);
```

Comment:
Do you expect there to be more common logic in this method? Otherwise I don't see the need for a separate `openObject`.